.github/workflows/maven.yml

```diff
@@ -26,5 +24,16 @@ jobs:
     - run: mkdir artifact && cp target/*.jar artifact
     - uses: actions/upload-artifact@v2
       with:
-       name: Package
-       path: artifact
+        name: os-server-${{ github.run_number }}.jar
+        path: os-server/target/os-server-**.jar
+    - name: Rename file
+      run: mv os-server/target/os-server-**.jar os-server/target/os-server-${{ github.run_number }}.jar
+    - name: Upload to S3 as artifact
+      uses: medlypharmacy/s3-artifacts-action@master
+      with:
+        aws_access_key_id: ${{ secrets.AWS_S3_ACCESS_ID}}
+        aws_secret_access_key: ${{ secrets.AWS_S3_ACCESS_KEY}}
+        aws_s3_bucket_name: ${{ secrets.AWS_S3_BUCKET_NAME }}
+        source_path: 'os-server/target/os-server-${{ github.run_number }}.jar'
```

Comment:
source path will be different for you I think

Reply:
The source path is changed